boards/amd/amd_acp_6_0_adsp/amd_acp_6_0_adsp.yaml

```diff
@@ -0,0 +1,9 @@
+identifier: acp_acp_6_0_adsp/acp_6_0
```

Comment:
this is not how you do board revisions, and why does the board and soc need revisions?